packages/react-menu/config/normalize-import.js

```diff
@@ -0,0 +1,96 @@
+// @ts-check
+
+/// <reference lib="es2019.array" />
+
+const fs = require('fs');
+const path = require('path');
+const { default: chalk } = require('chalk');
+const yargsParser = require('yargs-parser');
+
+/**
+ * @type {Console['log']}
+ */
+// eslint-disable-next-line no-console
+const log = console.log.bind(console);
+
+const scope = '@fluentui';
+
+main();
+
+/**
+ * Normalize cross package type imports in generated ambient types
+ *
+ * NOTE:
+ * This is a temporary workaround to TypeScript bug that occurs when path aliases are used
+ * For more info see: https://github.com/microsoft/TypeScript/issues/42349
+ */
+function main() {
+  /**
+   * @type {import('yargs').Arguments<{output?:string}>}
+   */
+  const args = yargsParser(process.argv.slice(2));
+
+  if (!args.output) {
+    throw new Error(`--output is missing`);
+  }
+
+  const sourceDir = path.resolve(process.cwd(), args.output);
+
+  if (!fs.existsSync(sourceDir)) {
+    throw new Error(`${sourceDir} doesn't exist`);
+  }
+
+  if (!isDirectory(sourceDir)) {
+    throw new Error(`${sourceDir} is not a directory`);
+  }
+
+  log(chalk.blue('Normalize ambient declarations...'));
+
+  const files = getAllFiles(sourceDir);
+  normalizeImports(files);
+
+  log(chalk.blue('Normalize ambient declarations: ✅'));
+}
+
+/**
+ *
+ * @param {string[]} files
+ */
+function normalizeImports(files) {
+  const regex = /(\.\.\/)+([a-z-]+)\/src/g;
+  files.forEach(filePath => {
+    const content = fs.readFileSync(filePath, 'utf-8');
+    const updatedContent = content.replace(regex, `${scope}/$2`);
+    fs.writeFileSync(filePath, updatedContent, 'utf-8');
+  });
+}
+
+/**
+ * Get all files absolute paths recursively in a directory
+ * @param {string} dirPath
+ * @param {string[]} arrayOfFiles
+ * @returns {string[]}
+ */
+function getAllFiles(dirPath, arrayOfFiles = []) {
+  const files = fs.readdirSync(dirPath);
+
+  const allFiles = files.map(file => {
+    const filePath = path.join(dirPath, '/', file);
+
+    if (isDirectory(filePath)) {
+      return getAllFiles(filePath, arrayOfFiles);
+    }
+
+    return filePath;
+  });
+
+  return allFiles.flat(Infinity);
```

Comment:
Using this means we've implicitly dropped support for Node 10. Practically speaking that's probably okay at this point (given that it will be officially unsupported after next month) but I'd prefer to do it separately with a more intentional/coordinated approach: updating the `@types/node` version, updating docs, and announcing to contributors.

Reply:
- AFAIK we support node 12 and up
- there are no restrictions on node on yarn set by tooling (missing engines field completely) ( CI uses 12 :) )
- @ling1726 what node are you using (as you're the only consumer for now)?

We should address all of this in different PR/sprint

Reply:
version 12, as it says in our contributing guide

Reply:
and here's what is written in our official setup guide 
![image](https://user-images.githubusercontent.com/1223799/111771454-03891500-88ac-11eb-94bf-efc354de6d26.png)

Reply:
The ["verify your environment" section](https://github.com/microsoft/fluentui/wiki/Setup#verify-your-environment) goes into more detail about current requirements. 

> node -v: Should be 12.x.x or 14.x.x (where "x" is some number). 10.x should still work for now. Newer versions may not work.

Again, I don't think that dropping 10 is a problem at this point, I would just prefer for it to happen in a more coordinated manner.